project.clj

```diff
@@ -45,7 +45,7 @@
 
   :profiles
   {:provided
-   {:dependencies [[com.firebolt/metabase-core "1.50"]]}
+   {:dependencies [[com.firebolt/metabase-core "1.56"]]}
```

Comment:
This version has to match up with `-Dversion=` parameter on [L43](https://github.com/firebolt-db/metabase-firebolt-driver/pull/85/files#diff-274071745a4e2a04b647d79d500537e6dc13eee54f44d0426140026293701d1bR43)